### PR TITLE
Lab3

### PR DESCRIPTION
**Pull request by jamiguel23**

### CODE REVIEW

**Comment by jamiguel23** [comment]
Was able to get the modal populating but the photo won't render. WIP

File: src/App.js [hunk 1]
```diff
@@ -2,21 +2,58 @@ import React, { Component } from 'react';
...
-
-          <Main />
-
-          
         
-          <Footer />
+
+        <Main
+        data={data} 
+        chosenBeast={this.chosenBeast}
```
Comment (RivaD2) [inline src/App.js:47]: Good work so far on lab!

We can indent these lines here 👍

File: src/App.js [hunk 2]
```diff
@@ -2,21 +2,58 @@ import React, { Component } from 'react';
...
+        />
+        <SelectedBeast 
+        title={this.state.title}
+        image_url={this.state.image_url}
+        description = {this.state.description}
+        showModal={this.state.showModal}
+        closeModal={this.closeModal}
```
Comment (RivaD2) [inline src/App.js:54]: Indent lines here as well 👍

File: src/SelectedBeast.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    return (
+      <div>
+        <Modal show={this.props.showModal} onHide={this.props.closeModal} >
+          <Modal.Header>
+            <Modal.Title>{this.props.title}</Modal.Title>
+          </Modal.Header>
+          <Modal.Body>
+            {this.props.description}
+          </Modal.Body>
+          <Image src={this.props.image_url} alt={this.props.title} onclick={this.handleClick} className="img-fluid"/>
```
Comment (RivaD2) [inline src/SelectedBeast.js:17]: Props here inside `<Modal />` (title, image_url, and description) are showing as undefined. Why might that be? Double check how props are being passed to the component. Grab a TA to get ya goin if you can 👍

**Pull request merged**